pandas/tests/window/test_rolling_skew_kurt.py

```diff
@@ -25,6 +27,24 @@ def test_series(series, sp_func, roll_func):
     tm.assert_almost_equal(result.iloc[-1], compare_func(series[-50:]))
 
 
+@pytest.mark.parametrize("sp_func, roll_func", [["kurtosis", "kurt"], ["skew", "skew"]])
+def test_low_variance_series(sp_func, roll_func):
```

Comment:
Just add a comment linking to the issue for future contributors.

```suggestion
def test_low_variance_series(sp_func, roll_func):
    # GH#62946
```